VERSION

```diff
@@ -1 +1 @@
-1.3.0-dev
+1.2.2
```

Comment:
Did you mean to commit this?

Reply:
Yes I am only bumping the minor number, Since this is only a bugfix.

Reply:
Sure but usually releases in OCI go through a voting process and so on. Bit odd to include it in an ordinary PR.

Reply:
I have not been following any such procedure on the SELinux package.  I guess I could, but have not in the past.  Bottom line, I would like to get this PR Verified as fixing the issue and then vendored into runc, so we can fix it there.

Reply:
Sure, do whatever works for you.

The release procedures for all `opencontainers` projects (except I guess `selinux`) have all used the voting system we use for spec releases, but it's pretty obvious that's not a great model for non-spec projects. I'm hoping to correct this after we get `1.0.0-rc8` out.